### DISCUSSION TITLE
What other models are needed, I will do

### DISCUSSION THREAD

**Discussion by Alex-test-mod** [Babyhamsta/Aimmy#1363] [General]
I have a server with five 12gb 3080 rtx (👍

**Comment by tuntungi** [comment]
Can you do POLYGON(steam) model ?. I tried to make one by myself but I have skill issue with it and Don't know how.
They have image for their new ingame character model (on steam) maybe that'll make it easier to detect full body

**Reply by AL99117** [reply]
Can you help make  model of pubg and Varolante?
